README.md

```diff
@@ -1467,3 +1467,9 @@ We wish you a good studing :3
 ## Hello everyone!
 ### I'm [Vlad](https://github.com/MelmQuty) and I didn't find any typo in ur README(maybe because it's perfect:rocket:) but I'm gonna show your my cat.He is a little impudent.:octocat:
 ![cat](https://raw.githubusercontent.com/MelmQuty/cat/master/cat.jpg)
+
+***
```

Comment:
Hi! Good job! Why did you close previous one?

Reply:
Hi! I create new branch with out correcting mispelld

Reply:
For the future, it was not necessary. All such changes must be made in the same branch. You just add another commit to the same PR.